script.js

```diff
@@ -1,5 +1,202 @@
+  return array
+}
+
+var playerTurn = function(playerNumber,choice,playerArray){
+  if (playerNumber == 1 && choice == null){
+    player1diceResults = roll2Dice(player1diceResults)
+    myOutputValue = `Player ${playerNumber}, Here is your rolled dice result: <br><br>
+    dice 1 : ${player1diceResults[0]} <br><br>
+    dice 2 : ${player1diceResults[1]} <br><br>
+    Please choose the order of desired dice result. <br>
+    Type 1 and click enter if you wish for dice 1 to be the first number <br>
+    Type 2 and click enter if you wish for dice 2 to be the first number`;
+    return myOutputValue
+  }
+  else if (playerNumber == 2 && choice == null){
+    player2diceResults = roll2Dice(player2diceResults)
+    myOutputValue = `Player ${playerNumber}, Here is your rolled dice result: <br><br>
+    dice 1 : ${player2diceResults[0]} <br><br>
+    dice 2 : ${player2diceResults[1]} <br><br>
+    Please choose the order of desired dice result. <br>
+    Type 1 and click enter if you wish for dice 1 to be the first number <br>
+    Type 2 and click enter if you wish for dice 2 to be the first number`;
+    return myOutputValue
+  }
+  else if (choice == 1){
+    myOutputValue = `${playerNumber} have chose dice 1 to be the first in order, your result is ${playerArray[0]}${playerArray[1]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[0]}${playerArray[1]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+  else if (choice == 2){
+    myOutputValue = `${playerNumber} have chose dice 2 to be the first in order, your result is ${playerArray[1]}${playerArray[0]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[1]}${playerArray[0]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+}
+
+// 3 conditions: 
+// a > b
+// b < a
+// a == b
+var gameOutcome = function () {
+  if (Number(player1diceResults.join("")) > Number(player2diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player1} for winning!!! <br><br>
+    ${player1} BEATS ${player2} with a score of ${Number(player1diceResults.join(""))} against ${Number(player2diceResults.join(""))} <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  } 
+  else if (Number(player2diceResults.join("")) > Number(player1diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player2} for winning!!! <br><br>
+     ${player2} BEATS ${player1} with a score of ${Number(player2diceResults.join(""))} against ${Number(player1diceResults.join(""))}<br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+  else if (Number(player2diceResults.join("")) == Number(player1diceResults.join(""))){
+    myOutputValue = `Issa Draw lubba dubba dub dub <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+};
```

Comment:
this is good but consider storing the `Number(player2diceResults.join("")` and `Number(player1diceResults.join(""))` as a variable before using it in your conditionals and messages. This is for readability purpose and also, if you find a bug in the way you are calculating the score, you only need to change it in 2 places instead of 10 with the current way your code is. 

Also, since you are making use of early return, you do not need to nest the conditions in `else if` and you actually don't even need the last conditional check.

To demonstrate:
```
var gameOutcome = function () {
  var player1Score = Number(player1diceResults.join(""));
  var player2Score = Number(player2diceResults.join(""))
  var myOutputValue = "";
  if (player1Score > player2Score) {
    myOutputValue = `Congratulations ${player1} for winning!!! <br><br>
    ${player1} BEATS ${player2} with a score of ${player1Score} against ${player2Score} <br><br>
    Click enter to start another new game`;
    return myOutputValue;
  } 
  
  if (player2Score > player1Score) {
    myOutputValue = `Congratulations ${player2} for winning!!! <br><br>
     ${player2} BEATS ${player1} with a score of ${player2Score} against ${player1Score}<br><br>
    Click enter to start another new game`;
    return myOutputValue;
  }

  myOutputValue = `Issa Draw lubba dubba dub dub <br><br>
    Click enter to start another new game`;
  return myOutputValue;
};
```